include/enclave/outside/ecalls.h

```diff
@@ -46,34 +38,3 @@ extern "C"
       sgx_enclave_id_t enclave_id,
       faasm_sgx_status_t* ret_val);
 }
-
```

Comment:
Weird diffing by Github here, but all this removed chunk here has been copied to `EnclaveInterface.h`